rotkehlchen/tests/utils/polygon_pos.py

```diff
@@ -2,6 +2,7 @@
 from rotkehlchen.fval import FVal
 from rotkehlchen.types import SupportedBlockchain
 
+ALCHEMY_RPC_ENDPOINT = 'https://polygon-mainnet.g.alchemy.com/v2/uNdnI7_6XXc7ayswOxtd_RuTBGojJhIf'
```

Comment:
why this change? Where else is this used?

Reply:
No what I mean is what did you achieve with this change.Isn't it the same thing? I don't understand what you did here. You just put the endpoint out in a variable that's only used in this file. It makes no sense.

Reply:
It is used here https://github.com/rotki/rotki/pull/6285/files#diff-c9a958383afbcea23aa81d29d4ec9aebbdef01d05fddc3769d18fd3d0b9f2e8bR21